website/assets/styles/04-components/person-profile.scss

```diff
@@ -0,0 +1,10 @@
+.c-person-profile {
+  display: block;
+  clear: both;
+}
+.c-person-profile img {
+  width: 165px;
+  float: left;
```

Comment:
Wow, not seen one of these for years! 😂

Reply:
>TBH I will likely rewrite the CSS at some point. Don't be offended!

Yes please!!! Apparently I deleted the TODO which was literally "TODO: Rewrite all the CSS to not be terrible", but that was the intention!